### DISCUSSION TITLE
[ponzi alert] shared project ((The project was scammed - the end of project))

### DISCUSSION THREAD

**Discussion by micro73** [Blockunity/WarningList#15] [General]
پروژه shared

ادرس سایت : shared88.cc

**هشدار اسکم و کلاهبرداری - سیستم سوددهی پانزی و ادعاهای واهی**

ادعای مطرح شده :‌
دفتر مرکزی SHARED که در سال 2015 تأسیس شد در هامبورگ آلمان قرار دارد. این شرکت در حال حاضر یک شرکت حرفه ای در اروپا با قدرت فنی فوق العاده، فناوری اطلاعات هوشمند و خدمات ارزش افزوده اینترنتی است. پلتفرم SHARED به طور رسمی در سال 2020 وارد بازار خاورمیانه شد. این پلتفرم محصولی است که به خرید آنلاین هوشمند اختصاص داده شده است. ما بیش از 6 سال است که در تجارت الکترونیک فعالیت می کنیم. با روحیه یک گام در یک زمان، ما به بهبود رقابت پذیری پلت فرم و گسترش سهم بازار ادامه می دهیم. پلتفرم SHARED با بسیاری از پلتفرم های تجارت الکترونیک بین المللی روابط بلندمدت و پایداری را ایجاد کرده است. ، در حال حاضر به چهار شرکت بزرگ تجارت الکترونیک بین المللی (Amazon، eBay، Souq، Noon) خدمت می کند.

![6](https://user-images.githubusercontent.com/103103225/183280817-2869e9c8-02e0-4d7b-aee1-be62135360fd.jpg)
![7](https://user-images.githubusercontent.com/103103225/183280820-6a49e0f4-d0f6-40a2-bb49-d3994aec95c0.jpg)
![8](https://user-images.githubusercontent.com/103103225/183280821-ded46ca4-d9aa-4a2c-9a80-d6b673010452.jpg)

**بررسی های انجام شده :
صفحه اصلی سایت تنها دارای ثبت نام و ورود هست و هیچ شبکه اجتماعی،معرفی،هویت گردانندگان،آدرس و ... در سایت وجود ندارد
با بررسی whois سایت مشخص شد هویت گردانندگان مخفی شده – سایت در تاریخ 2022-05-08 تاسیس شده – فقط به دو زبان فارسی و انگلیسی میتوان مشاهده کرد.**

![4](https://user-images.githubusercontent.com/103103225/183280835-61e4424c-c3e1-4b35-a873-0b1d43a6dec5.jpg)
![5](https://user-images.githubusercontent.com/103103225/183280839-ab4b7c9a-35b6-484b-910c-c82640e623af.jpg)
![13](https://user-images.githubusercontent.com/103103225/183280840-bb186c13-0a33-4968-b677-cff1af886a34.jpg)

طبق بررسی در اسکم ادوایزر :( ترجمه ماشینی)
هویت صاحب وب سایت در WOIS پنهان است
رتبه ترانکو (چقدر ترافیک) نسبتاً پایین است
سرور سایت دارای چندین وب سایت دیگر با بازبینی کم است
این وب سایت به عنوان یک کلاهبرداری در Scamadviser رأی داده شده است
نظرات منفی برای این وب سایت شناسایی شد
سن این سایت (بسیار) جوان است.
سایت به ما اجازه تجزیه و تحلیل محتوای آن را نمی دهد

![1](https://user-images.githubusercontent.com/103103225/183280846-0f25c299-d3d3-4291-80c4-7e0214945718.jpg)
![2](https://user-images.githubusercontent.com/103103225/183280848-3559dba0-3ed4-41ca-96df-6b229d764825.jpg)

طبق ادعای انها دارای تاییدیه از (Amazon، eBay، Souq، Noon) این شرکت ها هستند

![14](https://user-images.githubusercontent.com/103103225/183280857-96fbb02d-86d4-4506-bf1d-d0d815b3832b.jpg)

که فقط یکی از انهارا بررسی کردیم با پشتیبانی شرکت امازون صحبت کردم و نتیجه رو در تصویر زیر مشاهده کنید که سایت رو اسکم معرفی کردند و همکاری رو تایید نکردند.

![15](https://user-images.githubusercontent.com/103103225/183280869-6869a1fc-48a4-48f8-b4fe-4890e9e85060.jpg)

هشدار اسکم و کلاهبرداری

توضیحات و مدارک بیشتر یافت شده از یکی از گروه های در حال فعالیت::

حقوقِ شما به usdt_تتر(هر یک usdt معادلِ یک دلار است) پرداخت میشود، بنابراین باید یک کیفِ پولِ ارزِ دیجیتال داشته باشید که البته بهتر است صرافی نیز باشد زیرا بتوانید به تومان تبدیل کنید، پیشنهاد ما به شما بیت پین است.
در چهار سطح میتوانید مشغول به کار شوید..به عنوانِ مثال برای vip(سطح) دو باید مبلغِ تضمینِ انجامِ کار را(216usdt) به مدتِ شش ماه نزد شرکت امانت بگذارید(بعد از شش ماه به شما برگردانده میشود)..درین سطح روزانه 10.80(ده دلار و هشتاد سنت) درآمد دارید..ماهانه 324 دلار.

![16](https://user-images.githubusercontent.com/103103225/183280875-91c254b4-49f8-4926-a158-b8e6bd1b08bd.jpg)
![17](https://user-images.githubusercontent.com/103103225/183280876-0375ab48-a9ad-42de-a5b0-bc4afdbf87f2.jpg)
![22](https://user-images.githubusercontent.com/103103225/183280878-ab6759e0-6c48-42f1-83de-272a9476ef57.jpg)

چند قسمت زیبای ماجرا تبلیغات گسترده و فعالیت بدون تحقیق بسیاری از مردم

![21](https://user-images.githubusercontent.com/103103225/183280890-a7363511-e16b-49d3-b0af-989d22910f18.jpg)
![22](https://user-images.githubusercontent.com/103103225/183280893-9971d664-8b06-4e51-9983-40c147333e48.jpg)

قسمت کوچکی از فعالیت افراد نااگاه :

![100](https://user-images.githubusercontent.com/103103225/183280904-16929a6b-fc43-41cd-9bfe-4d8aab81ed40.jpg)
![101](https://user-images.githubusercontent.com/103103225/183280907-d4adccf8-d6b1-4c3a-9054-28fc50310c74.jpg)
![FZdrna6UUAIzHvX](https://user-images.githubusercontent.com/103103225/183283912-d78c970c-418d-4be1-a900-69e2aaf0f391.jpg)

**هشدار اسکم و کلاهبرداری**

**Comment by micro73** [comment]
وب سایت های مرتبط با این پروژه

![27](https://user-images.githubusercontent.com/103103225/183283651-f52f2897-c612-40c3-be83-e38ce66c1465.jpg)


آدرس ولت shared88.cc
گردش امروز 166 هزار دلار
TRnV8w7trSfoFxBaweEzCtwRcQcJgqu58J

![29](https://user-images.githubusercontent.com/103103225/183283702-92c83126-755c-4a66-a003-f4d2f861627a.jpg)


در ضمن
مدعی شده که سایت آلمانیست، ولی زبان آلمانی ندارد.
به صورت پیش فرض روی زبان فارسی باز می شود.
سایت در حالت زبان انگلیسی، باز هم بنرها فارسیست.


![30](https://user-images.githubusercontent.com/103103225/183283709-2c93fe4a-9393-4515-b4a7-dc96fdde7044.jpg)
![31](https://user-images.githubusercontent.com/103103225/183283711-dae61d59-32c2-4c3a-a75f-40563c8a0b82.jpg)
![32](https://user-images.githubusercontent.com/103103225/183283712-ffd2e015-b34c-4573-a510-d3f24505cca9.jpg)

**Comment by micro73** [comment]
و اين هم پاسخ مكاتبه اي كه با امازون داشتم:

من كوين سي از تیم CSBA هستم. امیدوارم که حالتون خوب باشه
 
 من عکس ارسالی شما را بررسی کردم، لطفا هیچ اطلاعاتی به آنها ندهید.

 پس از تحقیقات بیشتر متوجه شدم که این وب سایت یک منبع اسپم است.![image](https://user-images.githubusercontent.com/103103225/183401936-d9175adb-305a-4181-a619-4b532283eab3.png)

**Comment by micro73** [comment]
فعالیت گسترده شبکه shared در مشهدو قم و ماکو



![16](https://user-images.githubusercontent.com/103103225/184087245-ed0f18a0-564b-4afe-9f98-a576952c4e4f.jpg)
![shared1](https://user-images.githubusercontent.com/103103225/184087248-34af0602-bcb7-47eb-b1ca-c56b10c2d47b.jpg)
![shared2](https://user-images.githubusercontent.com/103103225/184087249-7379e586-0557-4c5b-a977-12c23acad43d.jpg)
![11](https://user-images.githubusercontent.com/103103225/184087224-55b05170-2b77-4546-92fb-d8435614d397.jpg)
![12](https://user-images.githubusercontent.com/103103225/184087235-e01c5785-5026-43a4-ab91-5d811f17587a.jpg)
![13](https://user-images.githubusercontent.com/103103225/184087238-1931afdb-e6d9-4ae4-b9f5-26f1e094c824.jpg)
![14](https://user-images.githubusercontent.com/103103225/184087242-7353b427-2097-47b8-92cf-1d0322174b63.jpg)

**Comment by micro73** [comment]
ادامه تبلیغات و فعالیت ها در خوزستان و قم 

![17](https://user-images.githubusercontent.com/103103225/184090678-6778d6f6-cacf-4993-b2dd-4abcdfe29c90.jpg)
![18](https://user-images.githubusercontent.com/103103225/184090684-e76f436a-a34c-4991-ba35-b0090c8a6ee0.jpg)
![19](https://user-images.githubusercontent.com/103103225/184090689-1c036b69-5563-4f56-9caa-4fc1b07df991.jpg)

**Comment by micro73** [comment]
خبر ها حاکی از ان است این پروژه اسکم شد و پایان کار پروژه پانزی shared

**Comment by Blockunity** [comment]
در گروه های مالباخته Shared این پیام درحال انتشار برای اسکم دوباره و زدن بیشتر جیب مردم هست.
متاسافنه از ساده بودن افراد اعتماد میکنند و تایید اسکم رو با انداختن تقصیر بر دوش فردی نامشخص سعی در جلب اعتماد دارند.

*******
دوستان توجه کنید
اگه قبل از  2022/8/2 میلادی ثبت نام کردید میتونید از این آدرس به شیرد وصل بشید
افرادی که بعد این تاریخ ارتقا دادن یا ثبت نام کردن بی اثر بوده چون تیم الکس به بهانه آپدیت نرم افزار شئرد، پلتفرم تقلبی به ما ارائه داده اند و اینگونه تمام فعالیت های ما بعد از تاریخ مذکور در پلتفرم تقلبی انجام شده است. و هر پولی که به حساب شرکت واریز شده در واقع به حساب تیم الکس رفته است.
و اینگونه تمام فعالیت های ما در سایت شئرد که در حال حاضر با این آدرس باز میشود بی اثر بوده است.

https://www.bountybox.vip/login

اکنون به این آدرس بروید و با همان رمز قبلی خود وارد شوید. و برداشت بزنید.

*******
